solc/CommandLineInterface.cpp

```diff
@@ -1099,6 +1121,20 @@ General Information)").c_str(),
 	if (!checkMutuallyExclusive(m_args, g_argColor, g_argNoColor))
 		return false;
 
+	static vector<string> const conflictingWithLSP{
+		// TODO: I think there are much more (almost all) options that should not added when --lsp is present.
```

Comment:
Why not have a supported list instead? I think the only logically relevant option is the `--allow-paths`, right?

Reply:
Actually, that is a good question (open for ideas) of what command line options would make sense. Allow-paths can be passed via client configuration JSON object (I'm doing that one right now for EVM version and remapping).

Reply:
Inverted. Resolving then. (?)

Reply:
Well, could have input from others :)

Reply:
@axic can you explain me quickly what you meant with "input from others"? :)